design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
```

Comment:
We need UUID's as Federation between multiple systems is a possible option coming down the line. But we don't need to display the UUID directly to the user, it just needs to be there in the data protocol.